### ISSUE TITLE
Plot.voronoi mark ignores the target option?

### ISSUE THREAD

**Issue by MaxwellBo** [observablehq/plot#1258]
https://observablehq.com/@mjbo/voronoi-mark-does-not-propagate-target-property-to-a-elemen 

Given some data:

```javascript
const data = [
  {
    id: 1,
    date: new Date("2019"),
    href: "https://groundhog-day.com/predictions?year=2019"
  },
  {
    id: 2,
    date: new Date("2020"),
    href: "https://groundhog-day.com/predictions?year=2020"
  }
]
```
,
```javascript
Plot.plot({
  marks: [
    Plot.tickX(data, {
      x: "date",
      stroke: "id",
      strokeWidth: 10,
      href: "href",
      target: "_blank"
    })
  ]
})
``` 

generates ticks that have svg

```html
<g aria-label="tick" stroke-width="10"><a fill="inherit" xlink:href="https://groundhog-day.com/predictions?year=2019" target="() => &quot;_blank&quot;"><line x1="20" x2="20" y1="0.5" y2="30" stroke="rgb(35, 23, 27)"></line></a><a fill="inherit" xlink:href="https://groundhog-day.com/predictions?year=2020" target="() => &quot;_blank&quot;"><line x1="620" x2="620" y1="0.5" y2="30" stroke="rgb(144, 12, 0)"></line></a></g>
``` 

and

```javascript
Plot.plot({
  marks: [
    Plot.tickX(data, {
      x: "date",
      stroke: "id",
      strokeWidth: 10,
      href: "href",
      target: "_blank"
    })
  ]
})
``` 

generates cells that have svg

```html
<g aria-label="voronoi"><a fill="inherit" xlink:href="https://groundhog-day.com/predictions?year=2019"><path d="M20,30L20,0.5L320,0.5L320,30Z" fill="rgb(35, 23, 27)"></path></a><a fill="inherit" xlink:href="https://groundhog-day.com/predictions?year=2020"><path d="M620,0.5L620,30L320,30L320,0.5Z" fill="rgb(144, 12, 0)"></path></a></g>
```

**Comment by mbostock** [comment]
We don’t currently support setting the target property. There is no **target** option; there is only an **href** option.

https://github.com/observablehq/plot/blob/main/README.md#marks

I’ll retitle this issue as a feature request. In the meantime your options are either to set a base href, or to post-process the SVG generated by Plot to add the target attribute.

**Comment by MaxwellBo** [comment]
Ah, I see there's been some confusion on my reading of the docs - I now understand that while `href` is a channel, `target` is _not_ and thus cannot be configured with `target: () => "_blank"`. 

However, specifying it with `target: "_blank"` results in identical behavior: opening a new tab with the `tickX` mark, but causing an Observable iframe crash with `voronoi`.

Is it fair to read https://github.com/observablehq/plot/blob/main/README.md#marks :

```
All marks support the following style options:

[...]
target - link target (e.g., “_blank” for a new window); for use with the href channel
[...]
```

as inaccurate?

**Comment by mbostock** [comment]
Ah, sorry. I misremembered. You’re right; there is a **target** constant option (so, `target: "_blank"` should work, but `target: () => "_blank"` gets coerced to `"() => &quot;_blank&quot;"`, as you noted; that is intended).

I’ll investigate whether there’s a bug with the Plot.voronoi mark. Just to correct the top post, the second code snippet should be:

```js
Plot.plot({
  marks: [
    Plot.voronoi(data, {
      x: "date",
      fill: "id",
      href: "href",
      target: "_blank"
    })
  ]
})
```

**Comment by mbostock** [comment]
Thanks for the report. A fix is up in #1259 and will be included in the Plot release this week.
